cmd/worker/main.go

```diff
@@ -67,7 +67,7 @@ func handleMessage(ctx context.Context, msg *pubsub.Message, packagesBucket *blo
 		resultsBucket = resultsBucketOverride
 	}
 
-	log.Info("Got request",
+	log.Info(worker.GotRequestLogMsg,
```

Comment:
nit: can we move this entire logging operation to logging.go too? Then we can make `gotRequestLogMsg` private too.